package/Makefile

```diff
@@ -74,7 +74,7 @@ package_all: package_ground package_matlab extract_firmware
 
 extract_firmware: $(FLIGHTPKGNAME)
 	mkdir -p "$(PACKAGE_DIR)"/firmware
-	unzip -f "$(FLIGHTPKGNAME)" -d "$(PACKAGE_DIR)"/firmware
+	unzip -u "$(FLIGHTPKGNAME)" -d "$(PACKAGE_DIR)"/firmware
```

Comment:
-f means "freshen" aka create no new files, just update existing ones.  -u "update" is what we want-- overwrite files without fear.